hansel/.shed.yml

```diff
@@ -0,0 +1,7 @@
+categories: [Sequence Analysis]
+description: Heidelberg and Enteritidis SNP Elucidation
+name: bio_hansel
+owner: nml
```

Comment:
Not sure if you submitted it to the wrong repo :)
IUC repo would also be a good candidate.